src/pages/image/_id.vue

```diff
@@ -160,9 +160,17 @@ const VImageDetailsPage = {
       if (this.image.filetype) {
         this.imageType = this.image.filetype
       } else {
-        axios.head(event.target.src).then((res) => {
-          this.imageType = res.headers['content-type']
-        })
+        axios
+          .head(event.target.src)
+          .then((res) => {
+            this.imageType = res.headers['content-type']
+          })
+          .catch(() => {
+            /**
+             * Do nothing. This avoid the console warning "Uncaught (in promise) Error:
+             * Network Error" in Firefox in development mode.
+             */
```

Comment:
I don't see this error when I remove this `catch` :thinking:

Is there something specific we should do to handle this network error in actuality rather than ignoring it like we have been? I'm not sure what the effect of not having the image type defined is.

Reply:
I'm not sure what to do other than ignore but maybe it's better to hide it in the UI instead of showing "Unknown". Could be related to this Sentry issue too https://sentry.io/share/issue/a7dd1a0bf2304516a38ec5ac065d4049/ ?